src/Select.js

```diff
@@ -1481,6 +1506,16 @@ export default class Select extends Component<Props, State> {
     }
   }
 
+  renderLiveRegion () {
```

Comment:
if the input (and hence the select) is not focused, don't render the A11yText elements.